modules/browser/BrowserCapabilities.js

```diff
@@ -109,7 +121,7 @@ export default class BrowserCapabilities extends BrowserDetection {
         // FIXME bandwidth stats are currently not implemented for FF on our
         // side, but not sure if not possible ?
         return !this.isFirefox() && !this.isEdge()
-            && !this.isSafariWithWebrtc();
+            && !(this.isSafariWithWebrtc() && !this.isSafariWithVP8());
```

Comment:
Did safari 12.1 also implement proper BW stats? If so, I don't think the VP8 check here is correct. They may both rely on the version being 12.1, but the semantics are different. Can you please clarify this?

Reply:
I believe it does. They've made a hell of a lot of changes in WebKit's included webrtc library
https://trac.webkit.org/changeset/237075/webkit

Reply:
Cool. Then please check if the browser is safari and if the version is 12.1, since BW stats are really not related to VP8 support, they just happened to both be released in 12.1

Reply:
Implements, but no proper, it needs to be adapted. Lets do it in other PR.